File: src/server/models/raider.go
package models

type Raider struct {
	Id      int64  `db:"id" json:"id"`
	Name    string `db:"name" json:"name"`
	ClassId int64  `db:"class_id" json:"classId"`
	Level   int64  `db:"level" json:"level"`
	Rank    int64  `db:"rank" json:"rank"`
	RaceId  int64  `db:"race_id" json:"raceId"`
}

File: src/client/src/models/raider.ts
export class Raider {
    id: number;
    classId: number;
    name: string;
    rank: number;

    constructor(id: number, name: string, classId: number, rank: number) {
        this.name = name;
        this.id = id;
        this.classId = classId;
        this.rank = rank;
    }
}